### ISSUE TITLE
Extend Login Timeout

### ISSUE THREAD

**Issue by yukdumboobumm** [projectsend/projectsend#1086]
Is there an option I can change in one of the config files to extend the amount of time before admin is logged out. Current default is quick (something like 30 minutes) but I'd prefer 24 hours. I don't mind digging around code but I could use a hint about where to start. Thanks in advance

**Comment by ignacionelson** [comment]
Hi! This is a php setting.
Please check the accepted answer here.
https://stackoverflow.com/questions/8311320/how-to-change-the-session-timeout-in-php
Hope that helps!

**Comment by yukdumboobumm** [comment]
This answer did in fact point me in the right direction but didn't solve anything.  My instance was still logging out after 24 minutes.

From /includes/app.php it looks like you're already setting a pretty long session time. Maybe I'm wrong (I'm def not a php guy). I noticed that my console was throwing this error every time I logged in:
`“PHPSESSID” will be soon rejected because it has the “SameSite” attribute set to “None” or an invalid value, without the “secure” attribute`

I added 
`ini_set('session.cookie_secure',true);`
before session_start() is called. This seems to be working...

Edit to note:
cookie expiration is defined on line 187 in app.php 
`define('COOKIE_EXP_TIME', 60*60*24*30);
`
but it's not used anywhere else in the code.
